tests/baselines/reference/tsbuild/declarationEmit/when-declaration-file-is-referenced-through-triple-slash-but-uses-no-references.js

```diff
@@ -176,9 +176,6 @@ exports.getVar = getVar;
       ]
     },
     "exportedModulesMap": {
-      "../src/common/nominal.ts": [
```

Comment:
This shows how we have updated the signature to d.ts emit so any update to type.d.ts wont unnecessarily update files importing nominal.ts

Reply:
I don’t quite understand the expected behavior here or what’s being shown. nominal.ts exports a type that uses a global type declared in types.d.ts. A change to types.d.ts could cause a change to any file importing nominal.ts.

Reply:
This one is reverting the state as if the d.ts signature was computed for the nominal.ts : That is as if buildinfo was generated before #43314
Having said that exportedModulesMap is a list of files that are imported/referenced in .d.ts so any change to those files changes their d.ts as well indirectly.
So looking at this, it does look like type.d.ts is needed in nominal.ts so i will check whats wrong.. But whatever it is, it has been that way from start.. (#39645) adds this test and its same.